gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -258,7 +260,7 @@ def get_predetermined_qc(version: str = CURRENT_VERSION, test: bool = False):
     },
 )
 
-# Platform PCA loadings
+# Platform PCA loadings.
```

Comment:
This doesn't give any extra information since its the same as the variable name so I think we can remove this.
```suggestion
```